### PR TITLE
Deeper spotbugs checks

### PR DESCRIPTION
**Pull request by pushker001**
- [X] Make sure you are opening from a **topic/feature/bugfix branch** (right side) and not your main branch!
- [X] Ensure that the pull request title represents the desired changelog entry
- [X] Please describe what you did

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -15,6 +15,9 @@
 
         <properties>
                 <jenkins.version>2.346.3</jenkins.version>
+                <spotbugs.effort>Max</spotbugs.effort>
+                <spotbugs.failOnError>false</spotbugs.failOnError>
```
Comment (MarkEWaite) [inline pom.xml:19]: This setting is the default, so it does not need to be included in the change.
```suggestion
```

**Review by MarkEWaite — CHANGES_REQUESTED** [review]
Thanks for the pull request.

The spotbugs warning that is being reported is one that you can correct in the source code by removing the check for null.  See the [ci results](https://ci.jenkins.io/job/Plugins/job/cron_column-plugin/view/change-requests/job/PR-11/1/spotbugs/type.-1629582128/type.-1629582128/category.79242641/source.7541b59e-ba4c-4592-9b56-f796ecb56f69/#115) for more details

**Comment by pushker001** [comment]
> Thanks for the pull request.
> 
> The spotbugs warning that is being reported is one that you can correct in the source code by removing the check for null. See the [ci results](https://ci.jenkins.io/job/Plugins/job/cron_column-plugin/view/change-requests/job/PR-11/1/spotbugs/type.-1629582128/type.-1629582128/category.79242641/source.7541b59e-ba4c-4592-9b56-f796ecb56f69/#115) for more details

can I create a new PR to do this correction?

**Comment by MarkEWaite** [comment]
> > Thanks for the pull request.
> > The spotbugs warning that is being reported is one that you can correct in the source code by removing the check for null. See the [ci results](https://ci.jenkins.io/job/Plugins/job/cron_column-plugin/view/change-requests/job/PR-11/1/spotbugs/type.-1629582128/type.-1629582128/category.79242641/source.7541b59e-ba4c-4592-9b56-f796ecb56f69/#115) for more details
> 
> can I create a new PR to do this correction?

Yes, but I'm not sure why you would create a new PR for the correction.  As a maintainer, I won't accept this pull request because it fails the Jenkins continuous integration build.  That failure is due to a spotbugs issue.  You can correct that spotbugs issue by making a Java code change as indicated by the message in the Jenkins job and in the GitHub checks page that shows the exact problem to be fixed.

Since I won't accept this pull request without a fix for that spotbugs warning, why not make the change in this pull request instead of creating a new pull request?

**Comment by MarkEWaite** [comment]
Replaced by #12

**Pull request closed without merging**